.browserslistrc

```diff
@@ -1,5 +1,8 @@
 # Browsers that we support
-
-chrome >= 64
-safari >= 11
-firefox >= 58
+# Picked up for fabric 7.0 In April 2025 Browsers released jan 2021
+chrome >= 88
```

Comment:
I really wonder who still runs chrome this old
is it really worth supporthing 0.00001% of people?

Reply:
I think 4-5 years of browser is ok.
It is not done to support the 0.000001% of people, is to discover new JS and TS syntax and adopt it gradually.
TSC and Rollup can do just a part of the work, they do not care about what the canvas support and doesn't they care only for ecma script syntax. Support new browsers means being able to drop old canvas workaround, or use new text api for example. That's it.

Reply:
Regarding using TSC, when we started migrating to TS we didn't know we would keep TS. The new build system was necesssary anyway and rollup was decently fast to setup.
Using TSC or BABEL to transpile is not the end of the world for me, it can be reevaluated in the future, but i don't really care, it works.